ADR/0026-appstudio-object-labels.md

```diff
@@ -0,0 +1,36 @@
+# 26. Record architecture decisions
+
+Date: 2023-06-02
+
+## Status
+
+Accepted
+
+## Context
+
+There is a growing number of CRDs and other objects that together describe the application and its constituent parts. We need to use a consistent set of labels to describe to which
+part of the application certain supporting objects belong. This is not a functional requirement as such but a consistent set of labels supports troubleshooting efforts as well
```

Comment:
This also relates to https://issues.redhat.com/browse/RHTAP-785 requesting a unification of labels/annotations/finalizers. Would it make sense to expand this ADR to help add additional clarity? It has been suggested that this might be similar to a follow-on to ADR-10.

Reply:
Thanks for pointing this out. I think this is very much related to https://issues.redhat.com/browse/RHTAP-785. I think the naming of the finalizers falls under the troubleshooting umbrella, that is part of this ADR.

Reply:
As for the ADR-10, I'm not sure these two ADRs talk about the same thing. The labeling in ADR-10 is for the purpose of the monitoring of the resource usage, while this ADR is more about internal consistency and troubleshooting.

Reply:
I've replied in https://issues.redhat.com/browse/RHTAP-785?focusedId=22409052&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22409052 to explain why I think this ADR is still slightly different from what is being asked for there though.